api/v1beta1/flowcollector_types.go

```diff
@@ -146,11 +146,17 @@ type FlowCollectorIPFIX struct {
 	OVNKubernetes OVNKubernetesConfig `json:"ovnKubernetes,omitempty" mapstructure:"-"`
 }
 
-type FeatureConfigType string
+// Agent feature, can be one of:<br>
+// - `PKT_DROP`, to track packet drops.<br>
+// - `DNS_TRACKING`, to track specific information on DNS traffic.<br>
+// - `FLOW_RTT`, to track L4 latency. <i>Unsupported (*)</i><br>
+// +kubebuilder:validation:Enum:="PKT_DROP";"DNS_TRACKING";"FLOW_RTT"
+type AgentFeature string
 
 const (
-	ConfigEnabled  FeatureConfigType = "ENABLED"
-	ConfigDisabled FeatureConfigType = "DISABLED"
+	PktDrop     AgentFeature = "PKT_DROP"
+	DNSTracking AgentFeature = "DNS_TRACKING"
+	FlowRTT     AgentFeature = "FLOW_RTT"
```

Comment:
if I am not mistaken Strings need to use Pascal not snake case ?
```go
        PktDrop     AgentFeature = "PktDrop"
	DNSTracking AgentFeature = "DnsTacking"
	FlowRTT     AgentFeature = "FlowRtt"
```

Reply:
[You're right](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#discriminated-unions), idk why I had in mind the best practice was upper case

Reply:
done